CONTRIBUTING.md

```diff
@@ -215,6 +215,26 @@ and false.
 * If a PR includes a deprecation/breaking change, notification should be sent to the
   [envoy-announce](https://groups.google.com/forum/#!forum/envoy-announce) email list.
 
+# Adding new extensions
+
+For developers adding a new extension, one can take an existing extension as the starting point.
+
+Extension configuration should be located in a directory structure like
+`api/envoy/extensions/area/plugin/`, for example `api/envoy/extensions/tracers/zipkin/`
```

Comment:
Let's use other extension as examples other than tracers, they only have `v4alpha` which might be confusing.